src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck(String nameMachine) {
+        super(nameMachine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(getNameMachine() + " started working");
```

Comment:
The message printed should be more informative. It should indicate that a 'Truck' is the type of machine that started working. For example, 'Truck <name> started working.'